tests/fixtures/synthetic_hf_state.py

```diff
@@ -31,6 +32,7 @@ def make_lm_tokenizer(config: dict):
 
 
 def make_dummy_lm(model_name: str, max_position_embeddings: int, tokenizer):
+    pytest.importorskip('transformers')
```

Comment:
skipping here could be a cause as well of `tests/test_synthetic_hf_state.py` being skipped.